rpcs3/rpcs3qt/save_data_list_dialog.cpp

```diff
@@ -181,7 +179,7 @@ void save_data_list_dialog::UpdateList()
 	m_list->clearContents();
 	m_list->setRowCount(::narrow<int>(m_save_entries.size()));
 
-	QVariantMap currNotes = m_gui_settings->GetValue(gui::m_saveNotes).toMap();
+	const QVariantMap currNotes = m_gui_settings->GetValue(gui::m_saveNotes).toMap();
```

Comment:
@Megamouse if you made camelCase vars a snake_case in one place, maybe it's worth changing everything the same way.

Reply:
I didn't explicitly change stuff here on purpose.
Those things are just tiny details I change here and there when I'm looking for reference code.
But instead of reverting them I sometimes just commit them because ... why not.